velox/docs/configs.rst

```diff
@@ -64,6 +64,17 @@ Generic Configuration
      - integer
      - 32MB
      - Used for backpressure to block local exchange producers when the local exchange buffer reaches or exceeds this size.
+   * - exchange.enable_flow_control
+     - boolean
+     - false
+     - If true, enable flow control in exchange. This reduces network throughput, but will also reduce the memory
```

Comment:
The reader may wonder what does "flow control in exchange" mean? Would you add a bit more explanation?